controller/config.go

```diff
@@ -31,7 +32,7 @@ import (
 )
 
 func (c *SensorController) watchControllerConfigMap(ctx context.Context) (cache.Controller, error) {
-	c.log.Info("watching sensor controller config map updates")
+	log.Info("watching sensor controller config map updates")
```

Comment:
Can you rebase off of master again; now that the logging changes have been merged?